### ISSUE TITLE
Physics freezing

### ISSUE THREAD

**Issue by Malwaremation** [ValkyrienSkies/Valkyrien-Skies-2#676]
### This issue occurs when only Valkyrien Skies and addons are installed and no other mods

- [X] I have tested this issue and it occurs when no other mods are installed

### Minecraft Version

1.18

### Mod Loader

Fabric

### Issue description

When a ship is going too fast the physics freeze and i can't do anything to unfreeze it.

### Issue reproduction

1.- Make a ship
2.- Make it very fast
3.- Go fast

**Comment by McSnurtle** [comment]
I have experienced similar on forge. (disclaimer: I was using create: clockwork)

**Comment by blockninja124** [comment]
I don't believe this is an issue anymore. Please make a new issue post if you are still experiencing this.
